inference-engine/tests/functional/inference_engine/lp_transformations/interpolate_transformation.cpp

```diff
@@ -251,9 +251,9 @@ const std::vector<InterpolateTransformationTestValues> testValues {
         },
         {
             ngraph::element::u8,
-            {{ngraph::element::f32}, {-0.32f}, {0.1f}},
+            {{}, {}, {}},
             ngraph::element::u8,
-            {{}, {}, {}}
+            {{ngraph::element::f32}, {-0.32f}, {0.1f}}
```

Comment:
If mode is not "nearest" we will have f32 at the output of the Interpolate. So, we don't need Convert here. Please, remove `ngraph::element::f32`

Reply:
removed convert, thank you!

Reply:
@v-Golubev, seem like still can not make other mode transformation test pass. The [spec ](https://github.com/openvinotoolkit/openvino/blob/master/docs/ops/image/Interpolate_4.md)says output precision should be the the same as input precision, is there some implicit convert or unified precision is used?

Reply:
@chenhu-wang Issue is in the reference function builder. I missed this moment.
Could you please update `ngraph::builder::subgraph::InterpolateFunction::getReference` so that `precisionAfterOperation `is used there to set the precision of the Interpolate output?
For example, you can use `openvino\inference-engine\tests\ngraph_helpers\lpt_ngraph_functions\src\avg_pool_function.cpp::99`

Reply:
done, thank you!